debsources/app/doc/routes.py

```diff
@@ -0,0 +1,68 @@
+# Copyright (C) 2015  The Debdoc developers <info@doc.debian.net>.
```

Comment:
I spot a too aggressive search&replace here ;)
(same below in the copyright notice)